### PR TITLE
MAL-278 | Change email notification

### PR DESCRIPTION
**Pull request by Mitsuroseba**

### CODE REVIEW

**Comment by naebal** [comment]
Build comment file: 
--------------
===============
Build site installed at http://95.85.26.139/build42
Behat tests: standard file http://95.85.26.139/build42/build_reports/report.html

--------------

Refer to this link for build results (access rights to CI server needed): 
http://95.85.26.139:8080//job/malmo_PR_BUILDER/42/
Test PASSed.

**Comment by naebal** [comment]
Build comment file: 
--------------
===============
Build site installed at http://95.85.26.139/build43
Behat tests: standard file http://95.85.26.139/build43/build_reports/report.html

--------------

Refer to this link for build results (access rights to CI server needed): 
http://95.85.26.139:8080//job/malmo_PR_BUILDER/43/
Test PASSed.

File: drupal/profiles/mal/modules/custom/mal_rules_send_email_action/mal_rules_send_email_action.module [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+/**
+ * Implement hook_rules_action_info().
+ */
+function mal_rules_send_email_action_rules_action_info() {
+  return array(
+    'mal_rules_send_email_action' => array(
+      'label' => t('Send email to certain school editors'),
+      'group' => t('Custom'),
+      'parameter' => array(
+        //pass in parameters
```
Comment (temoor-gilmutdinov) [inline drupal/profiles/mal/modules/custom/mal_rules_send_email_action/mal_rules_send_email_action.module]: Whitespace, capitalize, dot in the end

File: drupal/profiles/mal/modules/custom/mal_rules_send_email_action/mal_rules_send_email_action.module [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+  // Now, actually send the mails.
+  $params = array(
+    'subject' => $subject,
+    'message' => $message,
+  );
+
+  // Set a unique key for this mail.
+  $name = isset($element->root()->name) ? $element->root()->name : 'unnamed';
+  $key = 'rules_action_mail_to_users_of_role_' . $name . '_' . $element->elementId();  $languages = language_list();
```
Comment (temoor-gilmutdinov) [inline drupal/profiles/mal/modules/custom/mal_rules_send_email_action/mal_rules_send_email_action.module]: New line for language variable.

**Comment by naebal** [comment]
Build comment file: 
--------------
===============
Build site installed at http://95.85.26.139/build44
Behat tests: standard file http://95.85.26.139/build44/build_reports/report.html

--------------

Refer to this link for build results (access rights to CI server needed): 
http://95.85.26.139:8080//job/malmo_PR_BUILDER/44/
Test PASSed.

**Comment by naebal** [comment]
Build comment file: 
--------------
===============
<strong>23</strong> JSHint: modules standard file http://95.85.26.139/build45/build_reports/modulesjshint.txt
<strong>6816</strong> JSHint: profile standard file http://95.85.26.139/build45/build_reports/profilejshint.txt
Build site installed at http://95.85.26.139/build45

--------------

Refer to this link for build results (access rights to CI server needed): 
http://95.85.26.139:8080//job/malmo_PR_BUILDER/45/

**Build Log**
*last 25 lines*

 ```
[...truncated 7829 lines...]
	at hudson.model.Run.execute(Run.java:1789)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:374)
Caused by: java.io.FileNotFoundException: https://api.github.com/repos/malmostad/vuxenforvaltningens_informationsportal/statuses/f42d78149f4fed0a48016854375967f26c37e2cd
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1676)
	at sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1674)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.net.www.protocol.http.HttpURLConnection.getChainedException(HttpURLConnection.java:1672)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1245)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:254)
	at org.kohsuke.github.Requester.parse(Requester.java:451)
	at org.kohsuke.github.Requester._to(Requester.java:224)
	... 13 more
Caused by: java.io.FileNotFoundException: https://api.github.com/repos/malmostad/vuxenforvaltningens_informationsportal/statuses/f42d78149f4fed0a48016854375967f26c37e2cd
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1625)
	at java.net.HttpURLConnection.getResponseCode(HttpURLConnection.java:468)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getResponseCode(HttpsURLConnectionImpl.java:338)
	at org.kohsuke.github.Requester.parse(Requester.java:447)
	... 14 more
```

Test FAILed.

**Pull request merged**